src/app/[locale]/(houseHold)/[houseHoldId]/edit/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { db } from "@/drizzle";
+import { HouseHoldTable } from "@/drizzle/schema";
+import { eq } from "drizzle-orm";
+import { validate } from "uuid";
+
+export default async function EditHouseHoldPage({
+  params,
+}: {
+  params: Promise<{ houseHoldId: string }>;
+}) {
+  const { houseHoldId } = await params;
+  const household = await getHouseHold(houseHoldId);
+  const link = `localhost:3000/en/${houseHoldId}/${household?.invite?.link}`;
```

Comment:
brakuje schemy (https)

wyciągnij to localhost:3000 do zmiennej środowiskowej

i nie hardcode'uj języka (to en powinno się automatycznie z aktualnego langa podstawić)